www/scipylib/building/windows.rst

```diff
@@ -5,374 +5,221 @@ Building From Source on Windows
-    arch=YOUR_ARCHITECTURE`` . This can take anywhere from 15 minutes to
-    several hours, depending on your setup. 
- 6. Execute ``make sanity_test arch=YOUR_ARCHITECTURE``  and hope that no tests
-    fail (the message ``[sanity_test] Error 1 (ignored)`` is to be expected). 
-
-Now copy the files ``libatlas.a``, ``libcblas.a``, ``libf77blas.a`` and
-``liblapack.a`` from ``ATLAS\lib\YOUR_ARCHITECTURE`` to a directory of your
-choice, for example ``C:\BLASLAPACKLIBS``. 
-
-Building LAPACK
+Microsoft Visual C++ (MSVC)
+###########################
+
+NumPy and SciPy both support MSVC as the C/C++ compiler extension modules for the official
+binary distribution of Python. However, make sure that you download the correct version!
+For example, Python 2.7.x is compiled with Visual Studio 2008, and Python 3.5.1 is compiled
+with Visual Studio 2015. If you are using Python 2.7.x, you should visit this link `here <https://www.microsoft.com/en-gb/download/details.aspx?id=44266>`__
+to download the **Microsoft Visual C++ Compiler for Python 2.7**. If you are using Python 3.4.x and Windows 7, you
+should visit this link `here <https://www.microsoft.com/en-us/download/details.aspx?id=8279>`__ and download the
+**Microsoft Windows SDK for Windows 7**. If you are using Python 3.5.x, you should obtain the compiler via their `Visual Studio`_
+offering and download the **Community Edition**. If none of these configurations match your own, you will need to
+use one of the other build options described above. Please be aware that this option does does not come with a Fortran compiler,
+and the only one currently known to be compatible with MSVC is the **Intel Fortran compiler (ifort)**, which itself
+is difficult to obtain as will be explained in the discussion about the :ref:`MKL Library`.
+
+.. _Visual Studio: https://www.visualstudio.com/
+
+Libraries
+---------
+
+As mentioned in the overview, certain libraries (math libraries to be specific) are necessary
+for a high performing NumPy and for building SciPy, and they are BLAS_ and LAPACK_. There are
+many options available, in particular for BLAS_, and we will discuss several of the options below.
+
+.. _BLAS: http://www.netlib.org/blas/
+
+.. _`MKL Library`:
+
+Intel Math Kernel Library (MKL)
+###############################
+
+Intel has provided its own implementations of BLAS_ and LAPACK_, and they are by far some
+of the best performing libraries for **both** NumPy and SciPy. Unfortunately, they are not free and
+also require their own Fortran compiler for these libraries to work. While it is possible to obtain
+the libraries for free via their Community License (you can click `here <https://software.intel.com/sites/campaigns/nest/>`__
+to learn and click `here <https://registrationcenter.intel.com/en/forms/?productid=2558&licensetype=2>`__ to register),
+it does not come with their Fortran compiler, **ifort**, which is necessary for building both the NumPy and SciPy libraries with MKL.
```

Comment:
So on Windows it doesn't need `icc`, like it does on Linux? That's probably worth mentioning explicitly.